code/obj/item/plants.dm

```diff
@@ -213,6 +213,11 @@
 	desc = "A bland but healthy cereal crop. Good source of fiber."
 	icon_state = "oat"
 
+/obj/item/plant/oat/salt
+	name = "oat"
+	desc = "A salty but healthy cereal crop. Just dont eat too much without water."
```

Comment:
```suggestion
	desc = "A salty but healthy cereal crop. Just don't eat too much without water."
```

Reply:
truth be told, i do most of my coding at night, when im tired